content/scalability/docs/index.md

```diff
@@ -0,0 +1,28 @@
+# EKS Scalability best practices
+This guide provides advice for scaling EKS clusters. The goal of scaling an EKS cluster is to maximize the amount of work a single cluster can perform. Using a single, large EKS cluster can reduce operational load compared to using multiple clusters, but it has trade-offs for things like multi-region deployments, tenant isolation, and cluster upgrades. In this document we will focus on how to achieve maximum scalability with a single cluster.
+
+## How to use this guide
+This guide is meant for developers and administrators responsible for creating and managing EKS clusters in AWS. It focuses on some generic Kubernetes scaling practices, but it does not have specifics for self-managed Kubernetes clusters or clusters that run outside of an AWS region with [EKS Anywhere](https://anywhere.eks.amazonaws.com/).
+
+Each topic has a brief overview, followed by recommendations and best practices for operating EKS clusters at scale. Topics do not need to be read in a particular order and recommendations should not be applied without testing and verifying they work in your clusters.
+
+## Understanding scaling dimensions
+Scalability is different from performance and [reliability](https://aws.github.io/aws-eks-best-practices/reliability/docs/), and all three should be considered when planning your cluster and workload needs. As clusters scale, they need to be monitored, but this guide will not cover monitoring best practices. EKS can scale to large sizes, but you will need to plan how you are going to scale a cluster beyond about 300 nodes or 5000 pods.
```

Comment:
Two comments -
1. nit: Is the word "about" typo/redundant?
2.  Curious how you landed on the number "300 nodes or 5000 pods"?

Reply:
Those numbers came from lots of conversations with engineers and support teams for where customers started running into problems that these suggestions help them solve.

Reply:
Sounds good. Maybe can you add that clarification to the sentence, so that the reader understands where we came up with that guidance.